flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -263,6 +263,30 @@ public class JobManagerOptions {
                     .withDescription(
                             "The maximum number of historical execution attempts kept in history.");
 
+    /**
+     * Flag indicating whether JobManager should load available Failure Enricher plugins at startup.
+     * An optional list of Failure Enricher names. If empty, NO enrichers will be started. If
+     * configured, only enrichers whose name (as returned by class.getName()) matches any of the
+     * names in the list will be started.
+     *
+     * <p>Example:
+     *
+     * <pre>{@code
+     * jobmanager.failure-enrichers = org.apache.flink.test.plugin.jar.failure.TypeFailureEnricher, org.apache.flink.runtime.failure.FailureEnricherUtilsTest$TestEnricher
+     *
+     * }</pre>
+     */
+    @Documentation.Section(Documentation.Sections.ALL_JOB_MANAGER)
+    public static final ConfigOption<String> FAILURE_ENRICHERS_LIST =
+            key("jobmanager.failure-enrichers")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "An optional list of failure enricher names."
```

Comment:
should we specify that it is a comma separated list?

Reply:
We provide an example as part of the doc [above](https://github.com/apache/flink/pull/22467/files/001efd3caa6d586eaceefd835b31822392199f07#diff-f8a0e490a4d768924f942a38f12ae02eb6c096cc008a396944050a889fd5420dR275)  so i believe we are good here